src/MEDS_DEV/models/cehrbert/_config.yaml

```diff
@@ -9,6 +9,17 @@ model_initialization_dir: ???
 predictions_fp: ${output_dir}/predictions.parquet
 split: "held_out"
 demo: false
+streaming: ???
```

Comment:
Will this not cause errors if it is run without these arguments?

Reply:
@mmcdermott  i was debating about, if left empty, it would default to non-streaming, perhaps it's better to always do streaming?

Reply:
Updated, i set streaming to true.